package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-editor-service",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
weird, it marks it as changed, but this is already the version in `master`

Reply:
huh. well, rebasing seems to have solved it